### PR TITLE
Brush up Conceal Spell feat

### PR DESCRIPTION
**Pull request by ammalagonc**
Also, lint

### CODE REVIEW

File: packs/feats/conceal-spell.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
                 "property": "description",
                 "value": [
                     {
-                        "text": "PF2E.SpecificRule.Spellshape.ConcealSpell"
+                        "text": "PF2E.SpecificRule.Spellshape.ConcealSpell.Effect"
```
Comment (simonward) [inline packs/feats/conceal-spell.json:52]: This might need to be migrated?

Comment (ammalagonc) [reply packs/feats/conceal-spell.json:52]: Hmm - how much of a hassle would this be?

Comment (ammalagonc) [reply packs/feats/conceal-spell.json:52]: Any alternatives on how to do this without having to go through a migration?

Comment (simonward) [reply packs/feats/conceal-spell.json:52]: Not too bad - you can look at some of the other migrations to see examples.

Comment (ammalagonc) [reply packs/feats/conceal-spell.json:52]: Can you point me where to look at?
Kinda new to the concept of migrations.

Thank you again for walking me through this

Comment (simonward) [reply packs/feats/conceal-spell.json:52]: You'll want a migration that `extends MigrationBase`. Take a look at Migration924JiuHuanDoa. You'll need to test it's working with `npm run migrate`.

Comment (simonward) [reply packs/feats/conceal-spell.json:52]: Also, `recursiveReplaceString() might be of some use.

**Pull request closed without merging**